Xamarin.Forms.Platform.iOS/CollectionView/UICollectionViewDelegateForwarder.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using Foundation;
+using UIKit;
+
+namespace Xamarin.Forms.Platform.iOS
+{
+	public class UICollectionViewDelegator : UICollectionViewDelegateFlowLayout
+	{
+		public ItemsViewLayout ItemsViewLayout { get; set; }
+		public SelectableItemsViewController SelectableItemsViewController { get; set; }
+
+		public override void WillDisplayCell(UICollectionView collectionView, UICollectionViewCell cell, NSIndexPath path)
+		{
+			ItemsViewLayout?.WillDisplayCell(collectionView, cell, path);
+		}
+
+		public override UIEdgeInsets GetInsetForSection(UICollectionView collectionView, UICollectionViewLayout layout,
+			nint section)
+		{
+			if (ItemsViewLayout == null)
+			{
+				return default(UIEdgeInsets);
+			}
+
+			return ItemsViewLayout.GetInsetForSection(collectionView, layout, section);
```

Comment:
more question marks - less code =)
```suggestion
			return ItemsViewLayout?.GetInsetForSection(collectionView, layout, section) ?? default(UIEdgeInsets);
```